ktor-features/ktor-velocity/jvm/src/io/ktor/velocity/Velocity.kt

```diff
@@ -32,7 +33,7 @@ public class VelocityContent(
 /**
  * Velocity ktor feature. Provides ability to respond with [VelocityContent] and [respondTemplate].
  */
-public class Velocity(private val engine: VelocityEngine) {
+public open class Velocity(private val engine: VelocityEngine) {
```

Comment:
I would like to avoid having concrete open classes. Is it possible to keep this class closed, but extract common code to some external entity/top-level function/abstract class at least?
Also, maybe we can deprecate this feature in favor of the new one, what do you think @e5l?